data/mysql/schema.sql

```diff
@@ -1,3 +1,9 @@
+-- @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+-- @deprecated since eZ Platform 2.5.0 and no longer used.
+-- Relying on data/mysql/schema.sql to install eZ Platform schema is deprecated since eZ Platform v2.5.0.
+-- The file was removed in eZ Platform v3.0.0.
```

Comment:
'was' => 'will be' ?

Reply:
Also 'this file'

Reply:
> 'was' => 'will be' ?

depends on POV, it's already been done ;)

Reply:
lol yeah. inception! :-D

Reply:
Could do as well 'is removed'